src/aaudio/AAudioExtensions.h

```diff
@@ -37,22 +38,80 @@ namespace oboe {
 
 typedef struct AAudioStreamStruct         AAudioStream;
 
+const static std::set<DeviceType> ALL_OUTPUT_DEVICE_TYPES = {
```

Comment:
What happens when the device types expand over time? I don't think we'll have issues with expanding list but I just wanted to confirm

Reply:
Then the list here should also be updated. Added some comment